### PR TITLE
update translate_text_with_glossary

### PR DESCRIPTION
**Pull request by takatsugu-kato**
Change translation to with a glossary translation.

### CODE REVIEW

**Comment by takatsugu-kato** [comment]
I signed it!

**Comment by gguuss** [comment]
I just ran the tests locally and am seeing the responses have changed:

```
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.6.6, pytest-5.0.1, py-1.8.0, pluggy-0.12.0 -- /usr/local/google/home/class/temp/python-docs-samples/translate/cloud-client/env3/bin/python3
cachedir: .pytest_cache
rootdir: /usr/local/google/home/class/temp/python-docs-samples, inifile: pytest.ini
plugins: flaky-3.6.0
collected 10 items

beta_snippets_test.py::test_translate_text FAILED                                                                                                                                                                                       [ 10%]
beta_snippets_test.py::test_batch_translate_text PASSED                                                                                                                                                                                 [ 20%]
beta_snippets_test.py::test_detect_language PASSED                                                                                                                                                                                      [ 30%]
beta_snippets_test.py::test_list_languages PASSED                                                                                                                                                                                       [ 40%]
beta_snippets_test.py::test_list_languages_with_target PASSED                                                                                                                                                                           [ 50%]
beta_snippets_test.py::test_create_glossary PASSED                                                                                                                                                                                      [ 60%]
beta_snippets_test.py::test_get_glossary PASSED                                                                                                                                                                                         [ 70%]
beta_snippets_test.py::test_list_glossary PASSED                                                                                                                                                                                        [ 80%]
beta_snippets_test.py::test_translate_text_with_glossary FAILED                                                                                                                                                                         [ 90%]
beta_snippets_test.py::test_delete_glossary PASSED                                                                                                                                                                                      [100%]

================================================================================================================== FAILURES ===================================================================================================================
_____________________________________________________________________________________________________________ test_translate_text _____________________________________________________________________________________________________________
Traceback (most recent call last):
  File "/usr/local/google/home/class/temp/python-docs-samples/translate/cloud-client/beta_snippets_test.py", line 66, in test_translate_text
    assert 'Zdravo svet' in out
AssertionError: assert 'Zdravo svet' in 'Translated Text: translated_text: "Pozdrav svijetu"\n\n'
______________________________________________________________________________________________________ test_translate_text_with_glossary ______________________________________________________________________________________________________
Traceback (most recent call last):
  File "/usr/local/google/home/class/temp/python-docs-samples/translate/cloud-client/beta_snippets_test.py", line 125, in test_translate_text_with_glossary
    assert 'direcciones' in out
AssertionError: assert 'direcciones' in 'translated_text: "indicaciones"\nglossary_config {\n  glossary: "projects/intense-wavelet-343/locations/us-central1/glossaries/must-start-with-letters-4f5e38e2-b302-11e9-bc75-f4939feac106"\n}\n\n'
==================================================================================================== 2 failed, 8 passed in 127.78 seconds =====================================================================================================
```

Are the tests passing for you?

File: translate/cloud-client/beta_snippets.py [hunk 1]
```diff
@@ -269,8 +269,8 @@ def translate_text_with_glossary(project_id, glossary_id, text):
         target_language_code='es',
         glossary_config=glossary_config)
 
-    for translation in result.translations:
-        print(translation)
+    for glossary_translation in result.glossary_translations:
```
Comment (gguuss) [inline translate/cloud-client/beta_snippets.py]: You will need to update the tests, I'm seeing test_translate_text failing.

Comment (takatsugu-kato) [reply translate/cloud-client/beta_snippets.py]: I fixed the tests!

**Comment by gguuss** [comment]
@takatsugu-kato Looks like you'll need to resolve some conflicts.

**Pull request closed without merging**